examples/dogfooding/src/index.ts

```diff
@@ -0,0 +1,201 @@
+import {
+  createLightNode,
+  createEncoder,
+  createDecoder,
+  DecodedMessage,
+  waitForRemotePeer,
+  DefaultPubsubTopic,
+  LightNode,
+} from "@waku/sdk";
+
+import { Type, Field } from "protobufjs";
+import {
+  TelemetryClient,
+  TelemetryPushFilter,
+  TelemetryType,
+} from "./telemetry_client";
+import { generateRandomNumber, hashNumber } from "./util";
+
+const DEFAULT_CONTENT_TOPIC = "/js-waku-examples/1/message-ratio/utf8";
+const TELEMETRY_URL = process.env.TELEMETRY_URL || "http://localhost:8080/waku-metrics";
+
+const ProtoSequencedMessage = new Type("SequencedMessage")
+  .add(new Field("hash", 1, "string"))
+  .add(new Field("total", 2, "uint64"))
+  .add(new Field("index", 3, "uint64"))
+  .add(new Field("sender", 4, "string"));
+
+const sequenceCompletedEvent = new CustomEvent("sequenceCompleted");
+const messageSentEvent = new CustomEvent("messageSent");
+
+const wakuNode = async (): Promise<LightNode> => {
+  return await createLightNode({
+    contentTopics: [DEFAULT_CONTENT_TOPIC],
+    defaultBootstrap: true,
+  });
+};
+
+export async function app(telemetryClient: TelemetryClient) {
+  const node = await wakuNode();
+  await node.start();
+
+  // TODO: https://github.com/waku-org/js-waku/issues/2079
+  // Dialing bootstrap peers right on start in order to have Filter subscription initiated properly
+  await node.dial("/dns4/node-01.do-ams3.waku.test.status.im/tcp/8000/wss");
+  await node.dial("/dns4/node-01.ac-cn-hongkong-c.waku.test.status.im/tcp/8000/wss");
+  await node.dial("/dns4/node-01.gc-us-central1-a.waku.test.status.im/tcp/8000/wss");
+  
+  await waitForRemotePeer(node);
+
+  const peerId = node.libp2p.peerId.toString();
+  const encoder = createEncoder({
+    contentTopic: DEFAULT_CONTENT_TOPIC,
+  });
+
+  const startLightPushSequence = async (
+    numMessages: number,
+    period: number = 3000
+  ) => {
+    const sequenceHash = await hashNumber(generateRandomNumber());
+    const sequenceTotal = numMessages;
+    let sequenceIndex = 0;
+
+    const sendMessage = async () => {
+      try {
+        const message = ProtoSequencedMessage.create({
+          hash: sequenceHash,
+          total: sequenceTotal,
+          index: sequenceIndex,
+          sender: peerId,
+        });
+        const payload = ProtoSequencedMessage.encode(message).finish();
+        const result = await node.lightPush.send(encoder, {
+          payload,
+          timestamp: new Date(),
+        });
+        console.log("light push successes: ", result.successes.length);
+        console.log("light push failures: ", result.failures.length);
+        if (result.successes.length > 0) {
+          // Push to telemetry client
+          telemetryClient.push<TelemetryPushFilter>([
```

Comment:
We need to report failures with a message of failure. 
What columns can be used for that @adklempner ?